src/shared/components/plots/ScatterPlot.tsx

```diff
@@ -41,8 +45,9 @@ export interface IScatterPlotProps<D extends IBaseScatterPlotData> {
         spearman: number;
     };
     showRegressionLine?:boolean;
-    logX?:boolean;
-    logY?:boolean;
+    logX?:IAxisLogScaleParams|undefined;
+    logY?:IAxisLogScaleParams|undefined;
+    excludeLimitValuesFromCalculation?:boolean;
```

Comment:
could you add a comment for this?

Reply:
Added.

Reply:
Thanks. now that I understand what it is, I think it would be better to change 'Calculation' to 'Correlation' in the name

Reply:
Name was updated to `excludeLimitValuesFromCalculation`.